cognite_toolkit/cognite_modules/examples/cdf_functions_dummy/default.config.yaml

```diff
@@ -0,0 +1,3 @@
+example_secret: 'mysecret'
```

Comment:
This should be an environmental variable, and not an example of putting a secret in plain text of a checked in file. 
```suggestion
# This will be injected from the environment variable in the `cdf-tk deploy` command.
example_secret: '${MY_SECRET}'
```

Reply:
Right... Good catch!